cluster/test-deploy/Makefile

```diff
@@ -49,3 +49,10 @@ ifndef TO
 endif
 	cd $(PROFILE)/ && ../../bin/ansible.sh ansible-playbook -e "openshift_test_repo=$(REPO)" -e "openshift_$(TYPE)_image=$(TO)" -e "openshift_$(TYPE)_base_image=$(FROM)" playbooks/$(TYPE)/openshift-cluster/build_image.yml
 .PHONY: image
+
+play: check-cluster
+ifndef PLAY
+	$(error Must set PLAY= to a playbook path)
+endif
+	cd $(PROFILE)/ && ../../bin/ansible.sh ansible-playbook playbooks/$(PLAY)
```

Comment:
What if i want to pass arguments?  Like -vv is the next one.  And then `-e`.  Where does it stop?

Reply:
So this was really to work around what seems to be an inability to sanely pass through args to a make target... otherwise I'd be doing `make WHAT=dan sh ansible-playbook ...`

Is there some way of achieving that I missed?

Reply:
I went ahead and took this out so we can get the docs in

Reply:
Sanely pass through the argument array to `ansible.sh`, I should say